tests/simulate/test_lammps.py

```diff
@@ -72,15 +71,13 @@ def test_initialize(self):
         op = relentless.simulate.lammps.InitializeFromFile(filename=file_)
         l = relentless.simulate.lammps.LAMMPS(operations=op, quiet=False)
         sim = l.run(ensemble=ens, potentials=pot, directory=self.directory)
-        pl = lammps.PyLammps(ptr=sim.lammps)
-        self.assertIsNotNone(pl.system)
```

Comment:
I'm not sure how this test is equivalent to what was here before. It might be good to do some assertion on `sim.lammps` first, before you then check if the number of atoms is nonzero.